idp-service/src/main/resources/migrations/scorecard/checks.json

```
@@ -197,9 +197,89 @@
         "value": "true"
       }
     ],
-    "expression": "harness.noTestsFailingInCiPipeline == 0",
+    "expression": "harness.noTestsFailingInCiPipeline == true",
     "tags": ["Software quality", "Development maturity"],
     "defaultBehaviour": "FAIL",
     "failMessage": "Tests are failing on CI Pipeline"
+  },
+  {
+    "id": "65009e1bc28a1a3f5607f80c",
+    "accountIdentifier": "__GLOBAL_ACCOUNT_ID__",
+    "identifier": "__HARNESS__712025a1-c192-4212-8c29-252a37a44225",
+    "name": "Is on call set",
+    "description": "checks if on call is set on PagerDuty",
+    "ruleStrategy": "ALL_OF",
+    "rules": [
+      {
+        "data_source_identifier": "pagerduty",
+        "data_point_identifier": "isOnCallSet",
+        "operator": "==",
+        "value": "true"
+      }
+    ],
+    "expression": "pagerduty.isOnCallSet == true",
+    "tags": ["DevOps maturity"],
+    "defaultBehaviour": "FAIL",
+    "failMessage": "On call is not set on PagerDuty"
+  },
+  {
+    "id": "65009bfec28a1a3f5607f809",
+    "accountIdentifier": "__GLOBAL_ACCOUNT_ID__",
+    "identifier": "__HARNESS__566ff4fa-1ba8-40bf-bd4a-fd1ddb9a9b21",
+    "name": "Is escalation policy set",
+    "description": "checks if escalation policy is set on PagerDuty",
+    "ruleStrategy": "ALL_OF",
+    "rules": [
+      {
+        "data_source_identifier": "pagerduty",
+        "data_point_identifier": "isEscalationPolicySet",
+        "operator": "==",
+        "value": "true"
+      }
+    ],
+    "expression": "pagerduty.isEscalationPolicySet == true",
+    "tags": ["DevOps maturity"],
+    "defaultBehaviour": "FAIL",
+    "failMessage": "Escalation policy is not set on PagerDuty"
+  },
+  {
+    "id": "6500ac528ea5a4798502f512",
+    "accountIdentifier": "__GLOBAL_ACCOUNT_ID__",
+    "identifier": "__HARNESS__c758266c-22b4-454f-83c4-bd205f0ea277",
+    "name": "Number of incidents in last thirty days is less than 20",
+    "description": "Checks if number of incidents in last thirty days is less than 20 on PagerDuty",
+    "ruleStrategy": "ALL_OF",
+    "rules": [
+      {
+        "data_source_identifier": "pagerduty",
+        "data_point_identifier": "noOfIncidentsInLastThirtyDays",
+        "operator": "<",
+        "value": "20"
+      }
+    ],
+    "expression": "pagerduty.noOfIncidentsInLastThirtyDays < 20",
+    "tags": ["DevOps maturity"],
+    "defaultBehaviour": "FAIL",
+    "failMessage": "Number of incidents in last thirty days is more than 20 on PagerDuty"
```

Just for consistency in using numbers. All words is also fine, but might be too long.

Also an idea for future will be to show what the actual number is, and not just that it's more than 20. :) @vikyathharekal @sandeepa-kv-jois 

```suggestion
    "failMessage": "Number of incidents in last 30 days is more than 20 on PagerDuty."
```